frontend/packages/topology/src/geom/Rect.ts

```diff
@@ -112,6 +112,13 @@ export default class Rect implements Translatable {
     return this;
   }
 
+  //
+  // Padding Format:  [all], [horizontal, vertical], [left, vertical, right], [left, top, right, bottom]
```

Comment:
Not sure what I was thinking when I did this.
Let's update to follow the CSS shorthand notation.

`[all], [vertical, horizontal], [top, horizontal, bottom], [top, right, bottom, left]`